resources/js/processes/modeler/components/inspector/ModelerAssetQuickCreate.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div
+    id="asset-quick-create"
+    class="my-2"
+  >
+    <a
+      class="asset-link"
+      @click="goToAsset"
+    >{{ $t("Create a new") }} {{ label }} <i class="fa fa-plus" /></a>
+  </div>
+</template>
+<script>
+import { kebabCase } from "lodash";
+
+const AssetTypes = Object.freeze({
+  SCREEN: "screen",
+  SCRIPT: "script",
+  DECISION_TABLE: "decision table",
+});
+
+const channel = new BroadcastChannel("assetCreation");
```

Comment:
You can read more about it, here https://developer.mozilla.org/en-US/docs/Web/API/Broadcast_Channel_API